README.md

```diff
@@ -18,52 +21,54 @@ APIs in Node.js!
 
 ## System Requirements
 
-* IBM® SDK for Node.js - z/OS® 14.15.1 or higher or IBM Open Enterprise SDK for Node.js 16.0 or higher
-* z/OS V2R3 or higher
-* ZOAU v1.1.0 or higher is required on the system.
-  * For more details, visit: 
-    https://www.ibm.com/docs/en/zoau/latest?topic=installing-configuring-zoa-utilities
+- IBM® SDK for Node.js - z/OS® 14.15.1 or higher or IBM Open Enterprise SDK for Node.js 16.0 or higher
+- z/OS V2R3 or higher
+- ZOAU v1.1.0 or higher is required on the system.
+  - [For more details, see the zoau documentation.](https://www.ibm.com/docs/en/zoau/latest?topic=installing-configuring-zoa-utilities)
 
 ## Build and Install
 
-* Before installing, [download and install IBM Open Enterprise SDK for Node.js](https://www.ibm.com/products/sdk-nodejs-compiler-zos).
+- Before installing, [download and install IBM Open Enterprise SDK for Node.js](https://www.ibm.com/products/sdk-nodejs-compiler-zos).
 
 ## Setup
 
-* The PATH and LIBPATH environment variables need to include the location of the ZOAU
+- The PATH and LIBPATH environment variables need to include the location of the ZOAU
 binaries and dlls, respectively.
-``` bash
+
+```shell
 export PATH=<path_to_zoau>/bin:$PATH
 export LIBPATH=<path_to_zoau>/lib:$LIBPATH
 ```
-For more details on setting up ZOAU, visit:
-https://www.ibm.com/docs/en/zoau/latest?topic=installing-configuring-zoa-utilities
+
+[For more details on setting up ZOAU, see the documentation.](https://www.ibm.com/docs/en/zoau/latest?topic=installing-configuring-zoa-utilities)
```

Comment:
Should the word _documention_ be a link rather than the entire sentence?

Reply:
That seems reasonable.